src/areaCalculator/Main.java

```diff
@@ -1,8 +1,37 @@
 package areaCalculator;
 
+class Rectangle {
+    private double lengthA;
+    private double lengthB;
+
+    public Rectangle(double lengthA, double lengthB) {
+        this.lengthA = lengthA;
+        this.lengthB = lengthB;
+    }
+
+    public double getLengthA() {
+        return lengthA;
+    }
+
+    public double getLengthB() {
+        return lengthB;
+    }
+}
+
+class AreaCalculator {
+
+    public void calculateAreaOfRectangle(Rectangle rectangle) {
+        double area = rectangle.getLengthA() * rectangle.getLengthB();
+        System.out.println("Area of rectangle (" + rectangle.getLengthA() + " x "
+                + rectangle.getLengthB() + ") = " + area);
+    }
+}
+
 public class Main {
 
     public static void main(String[] args) {
-
+        Rectangle rectangle = new Rectangle(15.2, 22.3);
+        AreaCalculator area = new AreaCalculator();
```

Comment:
From the name of the variable I would assume that it stores area, but it actually stores area calculator.

Reply:
fixed